test/NuGet.Clients.Tests/NuGet.VisualStudio.Implementation.Test/Extensibility/VsFrameworkCompatibilityTests.cs

```diff
@@ -144,7 +144,7 @@ public void VsFrameworkCompatibility_GetNearestWithWithEmptyFallbackList()
             // Arrange
             var target = new VsFrameworkCompatibility();
             var targetFramework = new FrameworkName(".NETFramework,Version=v4.5.1");
-            var fallbackTargetFrameworks = new FrameworkName[0];
```

Comment:
A worthy deletion since it's unused, but there are no other changes in the file related to the PR's title.

Reply:
Similar to other changes, it was to get rid of error in VS UI.

Reverting.